nullaway/src/test/java/com/uber/nullaway/tools/ErrorDisplay.java

```diff
@@ -50,16 +77,19 @@ public boolean equals(Object o) {
     }
     ErrorDisplay that = (ErrorDisplay) o;
     return type.equals(that.type)
-        // To increase readability, a shorter version of the actual message might be present in the
-        // expected output of tests.
```

Comment:
Why are we removing this comment?

Reply:
Thank you for the notice, added it back 
e9182dbff7a4e538ab0d598d78cd7a1ed43c933d